Google.PowerShell.IntegrationTests/SQL/SQL.GcSqlInstances.Tests.ps1

```diff
@@ -211,6 +211,33 @@ Describe "Import-GcSqlInstance" {
         { Import-GcSqlInstance "test-db2" "gs://gcsql-csharp-import-testing/testsql.gz" "newguestbook" } | Should Throw `
         "ERROR 1840 (HY000) at line 24: @@GLOBAL.GTID_PURGED can only be set when @@GLOBAL.GTID_EXECUTED is empty."
     }
+
+    It "should be able to import a local file" {
+        { Import-GcSqlInstance "test-db2" "$PSScriptRoot\testsql.csv" "newguestbook" "entries" -UploadLocalFile} | Should not Throw
+    }
+
+    It "should delete the bucket for a local file upon completion" {
+        $oldBuckets = Get-GcsBucket
+        { Import-GcSqlInstance "test-db2" "$PSScriptRoot\testsql.csv" "newguestbook" "entries" -UploadLocalFile} | Should not Throw
+        $newBuckets = Get-GcsBucket
+        $oldBuckets.Count | Should Be $newBuckets.Count
+    }
+
+    It "should delete the bucket for a local file upon a file error" {
+        $oldBuckets = Get-GcsBucket
+        { Import-GcSqlInstance "test-db2" "$PSScriptRoot\testsqll.csv" "newguestbook" "entries" -UploadLocalFile} | Should Throw `
+        "Could not find file '$PSScriptRoot\testsqll.csv'"
+        $newBuckets = Get-GcsBucket
+        $oldBuckets.Count | Should Be $newBuckets.Count
+    }
+
+    It "should delete the bucket for a local file upon a instance error" {
+        $oldBuckets = Get-GcsBucket
+        { Import-GcSqlInstance "test-db2" "$PSScriptRoot\testsql.csv" "newguestbook" "entriees" -UploadLocalFile} | Should Throw `
```

Comment:
I assume entriees just doesn't exist either? It seems like it should have a better name, like "tablenotexist".